pod/annotations.go

```diff
@@ -159,6 +159,14 @@ const (
 	// service configuration
 
 	AnnotationKeyServicePrefix = "service.netflix.com"
+
+	// sidecar configuration
+
+	AnnotationKeyPrefixSidecars                      = "titusParameter.agent.sidecars"
```

Comment:
As soon as my platform sidecars proto change is in, then I'll make the control-plane add more structured annotations on the "real" prefix, so don't get too used to this key prefix.

However, that will make it harder to add new random annotations we need.

Reply:
Just to make sure I understand, the string itself will change, but not the actual mechanics, right? (there will still be a prefix, it just won't start with `titusParameter`...)

Reply:
correct, the string it self will change.

But that second part, once I make this part of the Titus API and the titus API is setting annotations, if we come up with a new class of annotation we want to use for more data, we'll have to do this `titusParameter.agent.sidecars.newthing` trick again.
Not the worst, but just a weird mixed mode.
Even more reason to get the API pretty right before we merge and start using it.